docker/errors_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	"testing"
```

Comment:
If we are going to copy the code, we should also copy `registry/client/errors_test.go`, or any other relevant tests from the original repo.